Pod/Swiftline/Args/Args.swift

```diff
@@ -29,20 +29,26 @@ public class Args {
       return result
     }
     
-    var result = [String: String]()
-    let parsedArges = ArgsParser.parseFlags(all)
+    var parsedFlags = [String: String]()
+    let parsedArgs = ArgsParser.parseFlags(all)
     
-    parsedArges.0.forEach {
-      result[$0.argument.name] = $0.value ?? ""
+    parsedArgs.0.forEach {
+      parsedFlags[$0.argument.name] = $0.value ?? ""
     }
-    
-    cachedResults = ParsedArgs(flags: result, parameters: parsedArges.1)
+
+    var arguments = parsedArgs.1
+    let commandName = arguments.first!
```

Comment:
I'm still wondering a bit if this is totally safe. is there any way this `first` element is not going to be available?